helm/hpcc/values.schema.json

```diff
@@ -431,6 +431,11 @@
           "type": "object",
           "additionalProperties": { "type": "string" },
           "description": "Global component annotations, generated into all components"
+        },
+        "plugins": {
```

Comment:
this definition is under "globals", it should be under "definitions".
And this should be a reference to the definition just as it is under the engine schemas.